src/components/Unknown/Root/index.tsx

```diff
@@ -27,15 +28,10 @@ const Root: React.FC = () => {
   console.log(data);
 
   return (
-    <div>
-      <Button color="primary" variant="contained">
-        Press Me
-      </Button>
-      <Button color="secondary" variant="contained">
-        Press Me
-      </Button>
-      <InputBase />
-    </div>
+    <>
+      <Header />
```

Comment:
this and <Container className={styles.buildPageContainer} maxWidth="lg"> you shold move to Layout